kubetest/process/process.go

```diff
@@ -170,9 +170,16 @@ func (c *Control) FinishRunning(cmd *exec.Cmd) error {
 		select {
 		case <-sigChannel:
 			log.Printf("Killing %v(%v) after receiving signal", stepName, -cmd.Process.Pid)
-			if err := syscall.Kill(-cmd.Process.Pid, syscall.SIGKILL); err != nil {
+
+			pgid, err := syscall.Getpgid(cmd.Process.Pid)
+			if err != nil {
```

Comment:
should this be `err == nil`?

Reply:
When will this fail? Log the error?

Reply:
maybe when the pid does not exist anymore? add a log to print the error if happens